tnc/modem.py

```diff
@@ -1261,7 +1264,7 @@ def calculate_fft(self) -> None:
         whether the channel is "busy."
         """
         # Initialize channel_busy_delay counter
-        channel_busy_delay = 0
+        #channel_busy_delay = 0
```

Comment:
I'm not sure if this is causing a possible crash?

Reply:
No  ModemParam.channel_busy_delay is storing this value with this PR.  We need to be able to set delay in the codec2 detection (see above comment code section)